test/testframe.c

```diff
@@ -445,7 +447,7 @@ GetTestExpress(void)
         env_val = HDgetenv("HDF5TestExpress");
 
         if (env_val == NULL)
-            SetTestExpress(1);
+            SetTestExpress(3);
```

Comment:
I'm wondering now if for Autotools we should add a configure option instead of changing the default in testframe. The github actions could use that and the default could still be 3, but it would probably be more visible to users at build time rather than needing an environment variable to run all the tests.

Reply:
Certainly would be useful.

Reply:
After a short discussion, adding an option to autotools has implications beyond what CMake does. 
Added Add autotools configure option for TestExpress #1729